CHANGELOG.d/README.md

```diff
@@ -6,21 +6,40 @@ Maintainers: see update-changelog.hs for details of this process.
 
 Contributors: read on!
 
-When you are preparing a new PR, add a new file to this directory. The file
-should be named `{PREFIX}_{SLUG}.md`, where `{PREFIX}` is one of the following:
-* `breaking`: for breaking changes
-* `feature`: for new features
-* `fix`: for bug fixes
-* `internal`: for work that will not directly affect users of PureScript
-* `misc`: for anything else that needs to be logged
+Our guiding principle is that the changelog is a tool for users—people who
+depend on PureScript as a compiler or as a library—who are considering
+upgrading, or have recently upgraded, their PureScript compiler version. We ask
+that when making changes that such users might need to know about, you help
+them out by adding to our changelog.
+
+Work that doesn't change the compiler (such as updates to README.md) doesn't
+need a changelog entry. But keep in mind that even parts of the project like
+our CI workflow can introduce changes to the compiler we release.
+
+When you are preparing a new PR that does change the compiler, add a new file
+to this directory. The file should be named `{PREFIX}_{SLUG}.md`, where
+`{PREFIX}` is one of the following:
+* `breaking`: for breaking changes to the compiler, for which a user may need to do
+  work to their project before or immediately upon upgrading
+* `feature`: for new features, which might prevent a user from downgrading to an
+  earlier version
+* `fix`: for bug fixes, which might motivate a user to upgrade
+* `internal`: for work that is not expected to directly affect users; these
+  entries should usually be brief, but may serve as useful starting points for
+  investigations if a change ends up having unintended consequences
+
+(There is also a fifth prefix, `misc`. This is an escape hatch in case we have
+something that somehow doesn't fit in the above categories but that we want to
+include in the changelog, which frankly seems unlikely given how much of a
+catch-all `internal` is. We'll tell you if you should use this one.)
```

Comment:
Why don't we drop this completely (an undocumented feature) and only bring it up if we need this escape hatch? It seems like 'internal' might be good enough.

Reply:
I figure this is partly documentation for contributors but also partly documentation for us—I don't want to leave us wondering in the future whether there was an explicit plan for the ‘Other improvements’ section. But maybe there's a better place than this file for that institutional knowledge?

Also and conversely, this is kind of a consensus-building PR; I'm calling this out explicitly as somewhat vestigial so that other core members might see and say, no, there's more intent than that behind this section.

Reply:
I think maybe we used `misc` in the ancient past for what we'd call `internal` now. I can't think of anything we'd particularly need it for in the future, given the current guidelines.

Reply:
So, is there consensus here that we should drop `misc`?

Reply:
Again, is there consensus about dropping `misc` here?